<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveFileBoxClass</class>
 <widget class="QWidget" name="SaveFileBoxClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件保存设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <item>
    <widget class="QGroupBox" name="formatGroupBox">
     <property name="title">
      <string>文件格式</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_format">
      <item>
       <widget class="QLabel" name="totalLinesLabel">
        <property name="text">
         <string>总行数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="totalLinesEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="csvRadioButton">
        <property name="text">
         <string>CSV</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="txtRadioButton">
        <property name="text">
         <string>TXT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rawRadioButton">
        <property name="text">
         <string>RAW</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bmpRadioButton">
        <property name="text">
         <string>BMP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="saveOptionsGroupBox">
     <property name="title">
      <string>保存选项</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_options">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_path">
        <item>
         <widget class="QLabel" name="pathLabel">
          <property name="text">
           <string>保存路径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pathEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseFolderButton">
          <property name="text">
           <string>浏览...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_prefix">
        <item>
         <widget class="QLabel" name="prefixLabel">
          <property name="text">
           <string>文件前缀：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="prefixEdit">
          <property name="text">
           <string>capture</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="appendTimestampCheckBox">
        <property name="text">
         <string>文件名添加时间戳</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="createSubfolderCheckBox">
        <property name="text">
         <string>创建日期子文件夹</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="saveRangeGroupBox">
     <property name="title">
      <string>保存范围</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_range">
      <item>
       <widget class="QRadioButton" name="saveAllRadioButton">
        <property name="text">
         <string>保存所有内容到一个文件内</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_lines">
        <item>
         <widget class="QRadioButton" name="splitByLinesRadioButton">
          <property name="text">
           <string>每个文件保存</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="linesPerFileSpinBox">
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="linesLabel">
          <property name="text">
           <string>行</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="saveRangeRadioButton">
        <property name="text">
         <string>按范围保存</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="rangeFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_rangeOptions">
         <item>
          <widget class="QCheckBox" name="lineRangeCheckBox">
           <property name="text">
            <string>选择行数范围</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_lineRange">
           <item>
            <widget class="QLabel" name="fromLineLabel">
             <property name="text">
              <string>从</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="fromLineSpinBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lineLabel1">
             <property name="text">
              <string>行</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="toLineLabel">
             <property name="text">
              <string>到</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="toLineSpinBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000000</number>
             </property>
             <property name="value">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lineLabel2">
             <property name="text">
              <string>行</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="columnRangeCheckBox">
           <property name="text">
            <string>选择列数范围</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_columnRange">
           <item>
            <widget class="QLabel" name="fromColumnLabel">
             <property name="text">
              <string>从</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="fromColumnSpinBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="columnLabel1">
             <property name="text">
              <string>列</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="toColumnLabel">
             <property name="text">
              <string>到</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="toColumnSpinBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>16</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="columnLabel2">
             <property name="text">
              <string>列</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="displayOptionsGroupBox">
     <property name="title">
      <string>显示选项</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_display">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_bytesPerLine">
        <item>
         <widget class="QCheckBox" name="maxBytesPerLineCheckBox">
          <property name="text">
           <string>每行最大显示数量</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="bytesPerLineComboBox">
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>64</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>128</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="bytesLabel">
          <property name="text">
           <string>字节</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="fileSaveContainer">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_buttons">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>开始保存</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>